checkout.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog</title>
+    <!-- normalize linking -->
+    <link rel="stylesheet" href="./styles/normalize.css" />
+    <!-- stylesheet linking -->
+    <link rel="stylesheet" type="text/css" href="./styles/stylesheet.css">
+  </head>
+
+  <body>
+    <div class="wrapper" id="clearfix">
+      <header>
+        <h1>Adopt A Dog</h1>
+        <p>Seattle's premiere dog adoption agency</p>
+        <a href="#main-content" class="hidden">Skip to main content</a>
+        <img src="./images/cart-icon.svg" alt="Shopping Cart Icon" width="50" height="50">
+        <p>$1234.56</p>
+        <button onclick="document.location='./checkout.html'">Checkout</button>
+        <img src="./images/adoptadog-logo.svg" alt="Adopt A Dog Logo" width="200" height="100">
+        <nav class="main-menu">
+          <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li>/</li>
+            <li><a href="./dogs.html">Dogs</a></li>
+            <li>/</li>
+            <li><a href="./blog.html">Blog</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <article class="checkout" id="main-content">
+        <h2 id="main-content">Checkout</h2>
```

Comment:
Note that IDs should be unique and only used in one HTML tag. If you'd like to use main-content in multiple places, you would want to convert this to a class. This will pop up in the code validator as well.